src/js/descript.js

```diff
@@ -99,29 +121,39 @@
         return this;
     };
 
+    /**
+     * Removes specified scripts from the default container.
+     * @param scriptAttributes
+     * @returns {Descript}
+     */
     Descript.prototype.remove = function(scriptAttributes) {
         this._process(scriptAttributes);
 
         return this;
     };
 
+    /**
+     * Injects a script into the container specified by `containerName`. In terms of position, the
+     * injected script is added directly after the script defined in `scriptAttribute`. The injected
+     * script is added as an inline script.
+     * @param scriptName
+     * @param containerName
```

Comment:
You need to specify which container you want to inject into. It's not always the default. In fact, I would see you calling this once you've shaped your containers.